app/views/admin/court_applications/show.html.erb

```diff
@@ -54,4 +68,17 @@
 <% if @court_application.hearing %>
   <%= link_to 'Back to Hearing', admin_hearing_path(@court_application.hearing) %>
 <% end %>
-</div>
\ No newline at end of file
+</div>
+
+
+<script type="text/javascript">
+  document.addEventListener('DOMContentLoaded', function() {
+    const selectBox = document.getElementById('environment_select');
+
+    selectBox.addEventListener('change', function() {
+      document.querySelectorAll('[button_group="environment_based"]').forEach((element) => {
+        element.style.display = element.attributes.target_env.value == selectBox.value ? "block" : "none"
+      });
+    });
+  });
+</script>
```

Comment:
JS on html.erb does not sounds great,
not blocking but it'd be nice to have it in external js file.

Reply:
Yeah, the only other JS we seem to have is a similar `<script>` tag on the prosecution_cases template. If we get any more JS I think it'll be time to refactor out.